### PR TITLE
Pia 1471

### PR DESCRIPTION
**Pull request by zladzeyka**
* The POST which resolves the payment request now returns the updates payment request including the requester uri. See the documentation for details.
  https://pay-api.gini.net/documentation/#response-16

### CODE REVIEW

File: GiniPayApiLib/Classes/Documents/Payments/ResolvedPaymentRequest.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+//  GiniPayApiLib
+//
+//  Created by Nadya Karaban on 05.05.21.
+//
+import Foundation
+/**
+ Struct for payment request response
+ */
+public struct ResolvedPaymentRequest: Codable {
+    public var requesterUri: String?
```
Comment (AlexGabor) [inline GiniPayApiLib/Classes/Documents/Payments/ResolvedPaymentRequest.swift]: Here requsterUri should always exist, so it shouldn't be nullable

File: GiniPayApiLib/Tests/Classes/SessionManagerMock.swift [hunk 2]
```diff
@@ -16,6 +16,8 @@ final class SessionManagerMock: SessionManagerProtocol {
     static let compositeDocumentId = "826626a0-749f-11e2-bfd6-000000000000"
     static let paymentProviderId = "b09ef70a-490f-11eb-952e-9bc6f4646c57"
     static let paymentRequestId = "118edf41-102a-4b40-8753-df2f0634cb86"
+    static let paymentRequesterUri = "ginipay_test://paymentRequester"
```
Comment (AlexGabor) [inline GiniPayApiLib/Tests/Classes/SessionManagerMock.swift]: shouldn't be `ginipay-test`?

File: GiniPayApiLib/Classes/Documents/Payments/ResolvedPaymentRequest.swift (hunk 1 above)
Comment (zladzeyka) [reply GiniPayApiLib/Classes/Documents/Payments/ResolvedPaymentRequest.swift]: Thanks Alex

File: GiniPayApiLib/Tests/Classes/SessionManagerMock.swift (hunk 2 above)
Comment (zladzeyka) [reply GiniPayApiLib/Tests/Classes/SessionManagerMock.swift]: I've copied from the response example, but I'll change it..better to have the same pattern for all paymentRequesterUri, you're absolutely right

**Review by AlexGabor — APPROVED** [review]

**Pull request merged**